Gemfile

```diff
@@ -1,2 +1,8 @@
 source 'https://rubygems.org'
-gem 'github-pages', group: :jekyll_plugins
```

Comment:
Can we keep the dependency on github-pages. This makes it easy to assure ourselves that the site will look the same in production as it does on our personal computers.